Author: Michael Terry <michael.terry@canonical.com>
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Date:   Wed Mar 13 14:14:07 2013 +0000
Description: Enable running tests during the build.
 Point to the exact location of Qt5's qmake while building the tests.
 We still don't have qtchooser available ;)
Last-Update: 2015-02-10


---
 Makefile |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,4 @@
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 MKDIR  = mkdir -p
 prefix = /usr
 bindir = $(prefix)/bin
@@ -75,7 +76,8 @@ uninstall:
 	;; esac
 
 tests/auto/Makefile: tests/auto/auto.pro
-	cd tests/auto && $(QMAKE) -o Makefile auto.pro
+	cd tests/auto && /usr/lib/$(DEB_HOST_MULTIARCH)/qt5/bin/qmake -o Makefile auto.pro
+	cd tests/auto/qtchooser && /usr/lib/$(DEB_HOST_MULTIARCH)/qt5/bin/qmake -o Makefile qtchooser.pro
 
 check: tests/auto/Makefile
 	cd src/qtchooser && $(MAKE) check
